azure-functions/acbs-function/mappings/facility/helpers/get-loan-maximum-liability.js

```diff
@@ -6,8 +6,8 @@ const { to2Decimals } = require('../../../helpers/currency');
  * `Loan` (EWCS) = Disbursement amount * (UKEF cover percentage / 100)
  * `Bond` (BSS) = Amount
  * @param {Float} amount Facility UKEF exposure
- * @param {object} facility Facility
- * @param {object} dealType Deal type
+ * @param {Object} facility Facility
+ * @param {Object} dealType Deal type
```

Comment:
Appreciate this isn't your part of your changes but `dealType` is a string

Reply:
Good point @ttbarnes have updated alongside with additional types.